frontend/src/stories/components/BasicCourseSearch/CourseOverTimeBuildingsSearchForm.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+import CourseOverTimeSearchBuildingsForm from "main/components/BasicCourseSearch/CourseOverTimeBuildingsSearchForm";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+
+
+export default {
+  title: "components/BasicCourseSearch/CourseOverTimeBuildingsSearch",
+  component: CourseOverTimeSearchBuildingsForm,
+  parameters: {
+    mockData: [
+      {
+        url: '/api/systemInfo',
+        method: 'GET',
+        status: 200,
+        response: systemInfoFixtures.showingBothStartAndEndQtr
```

Comment:
I _think_ this might be better as `systemInfoFixtures.showingBoth`, on account of `showingBothStartAndEndQtr` apparently not being a thing at https://github.com/ucsb-cs156-s23/proj-courses-s23-7pm-3/blob/main/frontend/src/fixtures/systemInfoFixtures.js.
That being said, I've become unsure what exactly this mockData is even supposed to do, so feel free to contest.